www/src/components/repository/Repository.tsx

```diff
@@ -23,14 +28,23 @@ import { RepositorySideCar } from './RepositorySideCar'
 
 import { REPOSITORY_QUERY } from './queries'
 
+// To test weither the provided url param is a uuid or a name
+const uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i
+
```

Comment:
Since we're already using the `uuid` package, consider replacing with `uuid.validate()` instead. https://www.npmjs.com/package/uuid#uuidvalidatestr

I see we've also got `isUuid(val)` in `www/src/components/shell/validation.tsx`, but it seems to be matching for a different uuid version. You could also add your new validation in the same file and rename them with appropriate versions. Possibly both using `uuid.validate()` with `uuid.version()`

Reply:
Great suggestion!